src/types/redis_hash.h

```diff
@@ -61,6 +61,8 @@ class Hash : public SubKeyScanner {
   rocksdb::Status Scan(const Slice &user_key, const std::string &cursor, uint64_t limit,
                        const std::string &field_prefix, std::vector<std::string> *fields,
                        std::vector<std::string> *values = nullptr);
+  rocksdb::Status RandField(const Slice &user_key, std::vector<FieldValue> *field_values, int64_t l, bool noparmeter,
```

Comment:
```suggestion
  rocksdb::Status RandField(const Slice &user_key, int64_t command_count, bool noparmeter, std::vector<FieldValue> *field_values, 
```

Seems that input field should better before output fields